specification/billing/resource-manager/Microsoft.Billing/stable/2021-10-01/billingSubscription.json

```diff
@@ -612,131 +681,545 @@
           "readOnly": true,
           "$ref": "#/definitions/Amount"
         },
-        "billingProfileId": {
-          "description": "The ID of the billing profile to which the subscription is billed.",
-          "type": "string",
-          "readOnly": true
+        "nextBillingCycleDetails": {
+          "description": "Next billing cycle details of the subscription.",
+          "type": "object",
+          "$ref": "#/definitions/NextBillingCycleDetails"
         },
-        "billingProfileDisplayName": {
-          "description": "The name of the billing profile to which the subscription is billed.",
+        "offerId": {
+          "description": "The offer ID for the subscription. This field is only available for the Microsoft Online Services Program billing accounts.",
           "type": "string",
           "readOnly": true
         },
-        "costCenter": {
-          "description": "The cost center applied to the subscription.",
-          "type": "string"
-        },
-        "customerId": {
-          "description": "The ID of the customer for whom the subscription was created. The field is applicable only for Microsoft Partner Agreement billing account.",
+        "productCategory": {
+          "description": "Category of the subscription. Possible values include: AzureSupport, Hardware, ReservationOrder, SaaS, SavingsPlanOrder, Software, UsageBased, Other",
           "type": "string",
           "readOnly": true
         },
-        "customerDisplayName": {
-          "description": "The name of the customer for whom the subscription was created. The field is applicable only for Microsoft Partner Agreement billing account.",
+        "productType": {
+          "description": "Type of the product for which the subscription is purchased",
           "type": "string",
           "readOnly": true
         },
-        "invoiceSectionId": {
-          "description": "The ID of the invoice section to which the subscription is billed.",
-          "type": "string",
-          "readOnly": true
+        "productTypeId": {
+          "description": "Id of the product for which the subscription is purchased",
+          "type": "string"
         },
-        "invoiceSectionDisplayName": {
-          "description": "The name of the invoice section to which the subscription is billed.",
+        "purchaseDate": {
           "type": "string",
+          "format": "date-time",
+          "description": "Purchase date of the product in UTC time.",
           "readOnly": true
         },
+        "quantity": {
+          "description": "The number of licenses purchased for the subscription",
+          "type": "integer",
+          "format": "int64"
+        },
         "reseller": {
-          "description": "Reseller for this subscription.",
+          "description": "Reseller for this subscription. The fields is not available for Microsoft Partner Agreement billing accounts.",
```

Comment:
"The reseller for which the subscription is created. The field is available for Microsoft Partner Agreement billing accounts."